internal/runner/service_test.go

```diff
@@ -218,6 +219,65 @@ func Test_runnerService(t *testing.T) {
 		assert.EqualValues(t, 0, result.Responses[1].ExitCode.Value)
 	})
 
+	// The longest accepted line must not have more than 1024 bytes on macOS,
+	// including the new line character at the end. Any line longer results in ^G (BELL).
+	// It is possible to send more data, but it must be divided in 1024-byte chunks
```

Comment:
> So, does this mean that the client-side should split the stdin into 1024 byte chunks separated by new line? Is there any reason we couldn't handle that on the server upon receiving an InputData request?

It's definitely an option, but it may not give the expected result.

Overall, commands like `runme run my-cmd < input.json` are affected and we should advise to put `< input.json` inside the `my-cmd`.

I'll update this comment because it's 1024 on macOS, but on Linux it's 4096 by default. Check out https://github.com/stateful/runme/pull/140#discussion_r1112140881.